project/cmake/modules/FindPython.cmake

```diff
@@ -5,39 +5,34 @@
 # PYTHON_INCLUDE_DIRS - the python include directory
 # PYTHON_LIBRARIES - The python libraries
 
-if(PKG_CONFIG_FOUND AND (NOT CMAKE_CROSSCOMPILING OR APPLE))
-  pkg_check_modules(PYTHON python QUIET)
+if(PKG_CONFIG_FOUND)
+  pkg_check_modules(PC_PYTHON python >= 2.7 QUIET)
 endif()
 
-if(NOT PYTHON_FOUND)
-  if(CMAKE_CROSSCOMPILING)
-    find_program(PYTHON_EXECUTABLE python ONLY_CMAKE_FIND_ROOT_PATH)
-    find_library(PYTHON_LIBRARY NAMES python2.6 python2.7)
-    find_path(PYTHON_INCLUDE_DIRS NAMES Python.h PATHS ${DEPENDS_PATH}/include/python2.6 ${DEPENDS_PATH}/include/python2.7)
-    set(PYTHON_INCLUDE_DIR ${PYTHON_INCLUDE_DIRS} CACHE PATH "python include dir" FORCE)
-
-    find_library(FFI_LIBRARY ffi)
-    find_library(EXPAT_LIBRARY expat)
-    find_library(INTL_LIBRARY intl)
-
-    if(NOT CORE_SYSTEM_NAME STREQUAL android)
-        set(PYTHON_DEP_LIBRARIES -lpthread -ldl -lutil)
-    endif()
-
-    set(PYTHON_LIBRARIES ${PYTHON_LIBRARY} ${FFI_LIBRARY} ${EXPAT_LIBRARY} ${INTL_LIBRARY} ${PYTHON_DEP_LIBRARIES}
-        CACHE INTERNAL "python libraries" FORCE)
-  else()
-    find_package(PythonLibs 2.7 REQUIRED)
+find_program(PYTHON_EXECUTABLE python ONLY_CMAKE_FIND_ROOT_PATH)
+find_library(PYTHON_LIBRARY NAMES python2.7 PATHS ${PC_PYTHON_LIBDIR})
+find_path(PYTHON_INCLUDE_DIR NAMES Python.h PATHS ${PC_PYTHON_INCLUDE_DIRS} ${DEPENDS_PATH}/include/python2.7)
+
+if(CMAKE_CROSSCOMPILING)
+  find_library(FFI_LIBRARY ffi REQUIRED)
+  find_library(EXPAT_LIBRARY expat REQUIRED)
+  find_library(INTL_LIBRARY intl REQUIRED)
+  find_library(GMP_LIBRARY gmp REQUIRED)
```

Comment:
Is there a reason why we need GMP support for a cross-compiled python env? In buildroot and Openelec the check is reverted:

https://github.com/bkuhls/buildroot/blob/392aebf38605cc849b4f5295615fd44f1cf56507/package/kodi/0003-CMake-Remove-dependency-on-gmp-and-libintl-from-Find.patch

https://github.com/OpenELEC/OpenELEC.tv/blob/master/packages/mediacenter/kodi/patches/kodi-998.02-cmake-python.patch

Reply:
ping, this question is still valid for Leia.

Reply:
This is only used when building with our own depends system and should be totally irrelevant for buildroot. OE is outdated and LE certainly does not set KODI_DEPENDSBUILD